apps/angular/src/shared/pipes/comma-join.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+/** Transform empty value.*/
+@Pipe({
+	name: 'commaJoin',
+})
+export class CommaJoinPipe implements PipeTransform {
+
+	/** @inheritdoc */
+	public transform(value: unknown): string | unknown {
+		if (value instanceof Array) {
+			return value.join(', ');
+		}
+
+		return value;
+	}
```

Comment:
We can improve this pipe:
1. Let's allow any joiner for array elements. We can pass it as pipe argument.
2. Improve type safety of pipe. We can restrict our pipe to work with arrays at compile time instead of runtime.

My suggestion looks like this but you can adjust it as needed:
```suggestion
@Pipe({
	name: 'join',
})
export class JoinPipe implements PipeTransform {

	/** @inheritdoc */
	public transform<T>(value?: readonly T[], joiner = ', '): string {
		if (value) {
			return value.join(joiner);
		}

		return '';
	}
```
You can provide pipe argument like this:
```html
{{ items | join }}
{{ items | join: '|' }}
```
More about Angular pipes you can find [here](https://angular.io/guide/pipes).